spread.yaml

```diff
@@ -57,6 +57,24 @@ backends:
                 username: ubuntu
                 password: ubuntu
 
+    external-ubuntu-core:
+        type: adhoc
+        environment:
+            SPREAD_ADHOC_UC_ADDRESS: "$(HOST: echo ${SPREAD_ADHOC_UC_ADDRESS:-localhost:8022})"
+        allocate: |
+            ADDRESS $SPREAD_ADHOC_UC_ADDRESS
+        systems:
+            # FIXME: remove the ubuntu-core-16.04-64 dummy system once the runner handles tasks without systems assigned
+            - ubuntu-core-16.04-64:
+                username: test
+                password: ubuntu
+            - ubuntu-core-16-device-amd64:
```

Comment:
Ok, as Samuele commented previously currently there are tests that should not be executed on ready made all-snap systems, but are ok to be executed on the all-snap image created for `ubuntu-core-16-64`,  for instance `tests/main/ack`.